src/time.rs

```diff
@@ -12,17 +12,29 @@ use crate::runtime::FutureIntoCoroutine;
 #[pyfunction]
 /// mock_time takes as argument a DateTime (for FrozenTime), an int (for ShiftedTime) or None (for RealTime)
 pub(crate) fn mock_time(time: &PyAny) -> PyResult<()> {
-    use libparsec::types::MockedTime;
-    libparsec::types::DateTime::mock_time(if let Ok(dt) = time.extract::<DateTime>() {
-        MockedTime::FrozenTime(dt.0)
-    } else if let Ok(us) = time.extract::<i64>() {
-        MockedTime::ShiftedTime { microseconds: us }
-    } else if time.is_none() {
-        MockedTime::RealTime
-    } else {
-        return Err(PyTypeError::new_err("Invalid field time"));
-    });
-    Ok(())
+    #![allow(unused_variables)]
+
+    #[cfg(not(feature = "test-utils"))]
+    {
+        Err(pyo3::exceptions::PyRuntimeError::new_err(
+            "Test features are disabled !",
+        ))
+    }
```

Comment:
You could remove the `allow unused_variable`

```suggestion
    #[cfg(not(feature = "test-utils"))]
    {
        drop(time);
        Err(pyo3::exceptions::PyRuntimeError::new_err(
            "Test features are disabled !",
        ))
    }
```

Reply:
That's pretty elegant !

Reply:
Edit: drop doesn't work for references:
```
error: calls to `std::mem::drop` with a reference instead of an owned value. Dropping a reference does nothing
```

and for Copy trait:
```
error: calls to `std::mem::drop` with a value that implements `Copy`. Dropping a copy leaves the original intact
```

However using the good old `let _foo = foo;` did the trick \o/